### PR TITLE
changes made to more specific imports

### LINKED ISSUE #3602: change "from MDAnalysis import *" to more specific imports
## Expected behavior ##
Docs show examples following Python best practices, in particular with regard to avoiding blanket imports of the form
```python
# AVOID THIS:
from MDAnalysis import *
```

## Actual behavior ##

The examples on the [overview](https://docs.mdanalysis.org/stable/documentation_pages/overview.html) page contain multiple cases of 
```python
from MDAnalysis import *
```

## Desired solution ##
Replace `from MDAnalysis import *` with accessing the required classes and functions within the module. For example. instead of `u = Universe(PSF, DCD)` write
```python
import MDAnalysis as mda

u = mda.Universe(PSF, DCD)
```
This is also how we recommend importing MDAnalysis, so it's important to have this common pattern early in the docs.

### PR DESCRIPTION
**Pull request by tamandeeps**
Fixes #3602 
Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

File: package/doc/sphinx/source/documentation_pages/overview.rst [hunk 1]
```diff
@@ -116,14 +116,14 @@ elementary and fundamental object in MDAnalysis.
...
-  >>> print u.trajectory
+  >>> print (u.trajectory)
   < DCDReader '/..../MDAnalysis/tests/data/adk_dims.dcd' with 98 frames of 3341 atoms (0 fixed) >
 
 Finally, the :meth:`MDAnalysis.Universe.select_atoms` method generates a new
 :class:`~MDAnalysis.core.groups.AtomGroup` according to a selection criterion
 
   >>> calphas = u.select_atoms("name CA")
-  >>> print calphas
+  >>> print (calphas)
```
Comment (orbeckst) [inline package/doc/sphinx/source/documentation_pages/overview.rst]: please revert, covered in PR #3588

File: package/doc/sphinx/source/documentation_pages/overview.rst [hunk 2]
```diff
@@ -116,14 +116,14 @@ elementary and fundamental object in MDAnalysis.
 The :attr:`MDAnalysis.Universe.trajectory` attribute gives access to the coordinates
 over time::
 
-  >>> print u.trajectory
+  >>> print (u.trajectory)
```
Comment (orbeckst) [inline package/doc/sphinx/source/documentation_pages/overview.rst]: please revert, covered in PR #3588

File: package/doc/sphinx/source/documentation_pages/overview.rst [hunk 3]
```diff
@@ -104,7 +104,7 @@ The :class:`~MDAnalysis.core.universe.Universe` contains a number of important a
 the most important ones of which is
 :attr:`~MDAnalysis.core.universe.Universe.atoms`::
 
-  >>> print u.atoms
+  >>> print (u.atoms)
```
Comment (orbeckst) [inline package/doc/sphinx/source/documentation_pages/overview.rst]: please revert, covered in PR #3588

**Review by orbeckst — CHANGES_REQUESTED** [review]
The changes you made spot-on, well done.

I liked your initiative of fixing Python 2 syntax but because this is already fixed in PR #3588, which will be merged soon, please revert those changes and keep your PR focused on the specific issue.

Please also
- add your name to AUTHORS
- add your Github handle to the CHANGELOG, at the end of the list of contributors for 2.2.0

File: package/doc/sphinx/source/documentation_pages/overview.rst (hunk 1 above)
Comment (tamandeeps) [reply package/doc/sphinx/source/documentation_pages/overview.rst]: Yes sir , changes applied.

File: package/doc/sphinx/source/documentation_pages/overview.rst (hunk 2 above)
Comment (tamandeeps) [reply package/doc/sphinx/source/documentation_pages/overview.rst]: Done sir

File: package/doc/sphinx/source/documentation_pages/overview.rst (hunk 3 above)
Comment (tamandeeps) [reply package/doc/sphinx/source/documentation_pages/overview.rst]: Done sir.

**Comment by orbeckst** [comment]
@tamandeeps I'll have a look at the changes in a bit.

There's no need to use "Sir" or any other honorifics. Everybody in the MDAnalysis Project addresses each other by first name or chosen handle. There's more explanation in the recent mailing list post [GSOC and Outreachy applicants: How to be a Good Member of a Scientific Software Community](https://groups.google.com/g/mdnalysis-devel/c/gWkydNDuc9A/m/2DMR7Dg9BwAJ) (which was posted before you joined). Please have a look.

**Comment by orbeckst** [comment]
You completed your task, you still have to

*    add your name to AUTHORS
*    add your Github handle to the CHANGELOG, at the end of the list of contributors for 2.2.0

**Comment by tamandeeps** [comment]
> 

Actually , I was feeling shy to add names to the AUTHORS because first I wanted to make a very significant contribution to this project as this contribution which I made now is very small.

**Comment by tamandeeps** [comment]
> > 
> 
> Yes actually , I was feeling shy to add names to the AUTHORS because first I wanted to make a very significant contribution to this project as this contribution which I made now is very small.

I have made the following changes

**Comment by tamandeeps** [comment]
> You completed your task, you still have to
> 
> * add your name to AUTHORS
> * add your Github handle to the CHANGELOG, at the end of the list of contributors for 2.2.0

I made the following changes. Added my name to Authors and added my github handle to CHANGELOG

**Review by orbeckst — APPROVED** [review]

**Comment by orbeckst** [comment]
Well done, congratulations on your first merged PR!

**Comment by orbeckst** [comment]
> Actually , I was feeling shy to add names to the AUTHORS because first I wanted to make a very significant contribution to this project as this contribution which I made now is very small.

We are happy about and welcome _every_ contribution that improves MDAnalysis and one way to make this clear is to acknowledge each contributor by having their name included. We hope to see more from you in the future!!

**Pull request merged**